RevenueCatUI/Views/IntroEligibilityStateView.swift

```diff
@@ -49,7 +49,7 @@ struct IntroEligibilityStateView: View {
         } else {
             // Display text with intro offer as a backup to ensure layout does not change
             // when switching states.
-            return self.textWithNoIntroOffer.notEmpty ?? self.textWithIntroOffer ?? ""
+            return self.textWithNoIntroOffer ?? self.textWithIntroOffer ?? ""
```

Comment:
Same thing here with testing for empty still for safety?

Reply:
Ditto.